### PR TITLE
Silviu stan/sm tm 0eadd select a subscription plan

### PR DESCRIPTION
**Pull request by SvSilviu**

### CODE REVIEW

File: src/main/java/pages/SubscriptionPlanPage.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+public class SubscriptionPlanPage extends BasePage {
+
+    public SubscriptionPlanPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = "//*[@id=\"root\"]/div/div/div[1]/form/div[3]/a[1]")
+    private WebElement createAccountBtn;
+
+    public void clickOnCreateAccountBtn() {
```
Comment (95teo) [inline src/main/java/pages/SubscriptionPlanPage.java:16]: Please move methods after WebElements meaning that you first will put WebElements grouped and then the methods after WebElements. Thanks!

File: src/main/java/pages/SubscriptionPlanPage.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class SubscriptionPlanPage extends BasePage {
+
+    public SubscriptionPlanPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = "//*[@id=\"root\"]/div/div/div[1]/form/div[3]/a[1]")
```
Comment (95teo) [inline src/main/java/pages/SubscriptionPlanPage.java:13]: Can you please change inside strings for selectors the double quotes (") to single quote (') and remove escape characters(**** or /)? Escape characters (\ or /) might cause some issues on different operating systems. Thanks!
For example, @FindBy(xpath = "//*[@id=\"root\"]/div/div/div[1]/form/div[3]/a[1]") -> @FindBy(xpath = "//*[@id='root']/div/div/div[1]/form/div[3]/a[1]")

Applicable for all elements that have double quotes (") inside strings.

**Review by 95teo — CHANGES_REQUESTED** [review]

File: src/main/java/pages/SubscriptionPlanPage.java [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    @FindBy(xpath = "//*[@id='root']/div/div[2]/div[1]/div/div[2]/form/div[3]/div[2]/div/div/div/section/div/div[2]/div[2]/div[2]/a[1]")
+    private WebElement selectPlanclick;
+
+    @FindBy(xpath = "//*[@id='root']/div/div[2]/div[1]/div/div[2]/form/div[3]/div[2]/div/div/div/section/div/div[2]/div[1]/div[2]/a[2]")
+    private WebElement goldSubscription;
+
+    @FindBy(xpath = "//*[@id='root']/div/div[2]/div[1]/div/div[2]/form/div[3]/div[2]/div/div/div/section/div/div[2]/a")
+    private WebElement continueBtn;
+
+
```
Comment (95teo) [inline src/main/java/pages/SubscriptionPlanPage.java:79]: No need to have two blank lines here. Leave only one. Thanks!

File: src/test/java/subscriptionPlanWhenRegisteringTests/SubscriptionPlanTests.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        reportUtils.createTestCare(new Throwable().getStackTrace()[0].getMethodName());
+        reportUtils.addTestLog(Status.INFO, "Performing log");
+        SubscriptionPlanPage subscriptionPlanPage = new SubscriptionPlanPage(driver);
+        subscriptionPlanPage.insEmail();
+        subscriptionPlanPage.setInsertPassword();
+        subscriptionPlanPage.setEnterVirtualWorld();
+        subscriptionPlanPage.setMyAccout();
+        subscriptionPlanPage.setSelectMySubscriprions();
+        subscriptionPlanPage.setSelectPlan();
+
```
Comment (95teo) [inline src/test/java/subscriptionPlanWhenRegisteringTests/SubscriptionPlanTests.java:45]: No need to leave a blank line between last line of a method and }. You can remove it. Applicable for all tests. Thanks!
Same comment for first blank line inside the method.

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by 95teo — APPROVED** [review]

**Review by stanciuvlad99 — APPROVED** [review]

**Pull request merged**